README.md

```diff
@@ -1,29 +1,40 @@
-# SOLID FORMS
+# Solid SDK Forms
 
 <img src="https://solid.inrupt.com/themes/custom/solid/logo.svg" width="150" height="150">
 
-Solid Form project to handle forms using ShEx and SHACL.
+This is a standalone Typescript library designed to handle form conversion and other form-related features.
 
-### Features
+*NOTE*: This library is still a work in progress. We are using lessons learned while building the library to improve the code and structure.
 
- - Build form using ShEx
- - Build form using SHACL
+## Installation
 
-### NPM scripts
+To install, simply run the command
 
- - `npm t`: Run test suite
- - `npm start`: Run `npm run build` in watch mode
- - `npm run test:watch`: Run test suite in [interactive watch mode](http://facebook.github.io/jest/docs/cli.html#watch)
- - `npm run test:prod`: Run linting and generate coverage
- - `npm run build`: Generate bundles and typings, create docs
- - `npm run lint`: Lints code
- - `npm run commit`: Commit using conventional commit style ([husky](https://github.com/typicode/husky) will tell you to use it if you haven't :wink:)
+`npm install --save @inrupt/solid-sdk-forms`
 
-For a library, `core-js` plays very nicely, since you can import just the polyfills you need:
+## Usage
+
+There are a few things you can do with this library. The first and most common usage will be converting a shape to a Form Model object.
```

Comment:
There are a few things you can do with this library. The most common usage will be converting a ShEx shape to a Form Model object.